scripts/install.sh

```diff
@@ -13,6 +13,9 @@ printf 'Y\n' | apt-get install -y maven
 printf 'Y\n' | apt-get install git
 printf 'Y\n' | apt-get install npm
 
+#install bro
```

Comment:
Line-14 installs old version of npm. Line-48 installs the latest version of nodejs 6.x and a matching npm. Hence, we can remove line-14.